.github/workflows/release.yml

```diff
@@ -0,0 +1,160 @@
+---
+# When a tag is created
+# - create a new release from the tag
+# - build and attach packages to the release
+name: Create packages and release
+
+on:
+  push:
+    tags:
+      - "v*"
+    branches:
+      - "main"
+
+jobs:
+  centos7:
```

Comment:
Also could avoid duplication here, but I'm not sure on how this is dealt with the `${{ steps.package_name_centos7.outputs.rpm_path }}` below

Reply:
I've currently no idea how to handle this, I would certainly be nicer to use a matrix too, but I guess we could postpone this, to at least move forward with the main stuff.